python_modules/dagster/dagster/_core/definitions/asset_layer.py

```diff
@@ -644,6 +644,15 @@ def asset_checks_def_for_node(
     def asset_checks_defs(self) -> Iterable[AssetChecksDefinition]:
         return self.asset_checks_defs_by_node_handle.values()
 
+    def get_asset_check_for_output_name(self, output_name: str) -> Optional[AssetCheckHandle]:
```

Comment:
I don't think output names are unique across the asset layer

Reply:
instead I would use `check_handle_by_node_output_handle`, looks like you could get a node output handle in the callsite